src/app/models/types/global.ts

```diff
@@ -18,7 +18,8 @@ export type PageTypes =
   | 'STY'
   | 'PGL'
   | 'CSP'
-  | 'TOPIC';
+  | 'TOPIC'
+  | 'HOME';
```

Comment:
I think when this goes live we will be using the existing `frontPage` as the type for this page type. Worth checking with Johann though.